src/main/scala/com/redislabs/provider/redis/RedisConfig.scala

```diff
@@ -43,6 +43,24 @@ case class RedisEndpoint(host: String = Protocol.DEFAULT_HOST,
     )
   }
 
+
+  /**
+   * Constructor from spark config and parameters.
+   *
+   * @param conf spark context config
+   * @param parameters source specific parameters
+   */
+  def this(conf: SparkConf, parameters: Map[String, String]) {
```

Comment:
Could we also update [structured streaming docs](https://github.com/RedisLabs/spark-redis/blob/master/doc/structured-streaming.md) similar to the [DataFrame options](https://github.com/RedisLabs/spark-redis/blob/master/doc/dataframe.md#dataframe-options) please?

Reply:
@fe2s Done..